content/sql/concepts/aggregate-functions/terms/count/count.md

```diff
@@ -51,3 +91,5 @@ The result would be:
 | COUNT(\*) |
 | --------- |
 | 2         |
+
+**Note:** NULL value is excluded due to the `WHERE experience < 5` command. 
```

Comment:
Let's take this out of a note since it's not off topic to the example above.

Reply:
@yangc95 
Thank you for your feedback.  I removed this example from the document.